content/contribute/_index.md

```diff
@@ -9,39 +9,37 @@ weight: 80
 
 We'd love your help!
 
-BotKube is [MIT Licensed](LICENSE) and accepts contributions via GitHub pull requests. This document outlines some of the conventions on development workflow, commit message formatting, contact points and other resources to make it easier to get your contributions accepted.
+BotKube is [MIT Licensed](/LICENSE) and accepts contributions via GitHub pull requests. This document outlines some of the conventions on development workflow, commit message formatting, contact points and other resources to make it easier to get your contributions accepted.
 
 We gratefully welcome improvements to [documentation](https://www.botkube.io/ "Go to documentation site") as well as to code.
 
 ## Contributing to documentation
 
-You can contribute to documentation by following [these instructions](https://github.com/infracloudio/botkube-docs#contributing "Contributing to BotKube Docs")
+You can contribute to documentation by following [these instructions](https://github.com/kubeshop/botkube-docs#contributing "Contributing to BotKube Docs")
 
 ## Compile BotKube from source code
 
-Before you proceed, make sure you have installed BotKube Slack/Mattermost/Teams app and copied required token as per the steps documented [here](https://www.botkube.io/installation/)
+Before you proceed, make sure you have installed BotKube Slack/Mattermost/Teams app and copied the required token as per the steps documented [here](https://www.botkube.io/installation/)

```

Comment:
Nope, the search doesn't work because there is a `baseURL = "https://www.botkube.io/"` in config.toml and the `index.json` is a built-in Hugo functionality and it's served on baseURL.

---

I updated the https://www.botkube.io and https://botkube.io